Makefile

```diff
@@ -146,5 +145,6 @@ validate-local:
 test-unit-local:
 	$(GPGME_ENV) $(GO) test -tags "$(BUILDTAGS)" $$($(GO) list -tags "$(BUILDTAGS)" -e ./... | grep -v '^github\.com/containers/skopeo/\(integration\|vendor/.*\)$$')
 
+.PHONY: vendor
```

Comment:
Why add this in two places?

Reply:
no one is enough :-) I refactored it but forgot to remove the old one